libpod/container_validate.go

```diff
@@ -66,7 +66,7 @@ func (c *Container) validate() error {
 
 	// Rootless has some requirements, compared to networks.
 	if rootless.IsRootless() {
-		if len(c.config.Networks) > 0 {
+		if !c.config.NetMode.IsSlirp4netns() && len(c.config.Networks) > 0 {
```

Comment:
I think CNI networks are still unsupported with Slirp, so this should probably be removed unless I missed something

Reply:
Unfortunately without this, a any param to --net that contains a : character results in an error about rootless not supporting choosing a network CNI

Reply:
Hmmm. Let me look a bit, but I suspect this is mostly because of how we're parsing at the command line level.